cibuildwheel/macos.py

```diff
@@ -1,12 +1,16 @@
 import os
+import platform
 import shlex
 import shutil
 import subprocess
 import sys
 import tempfile
 from pathlib import Path
-from typing import Dict, List, NamedTuple, Optional, Sequence
+from typing import Any, Dict, List, NamedTuple, Optional, Sequence, Set, Tuple, cast
 
+from typing_extensions import Literal
```

Comment:
This import fails when using the GHA action: https://github.com/mayeut/pybase64/runs/1797295461?check_suite_focus=true

Reply:
strange.. it's listed in setup.cfg as a requirement... https://github.com/joerick/cibuildwheel/blob/macos-universal2/setup.cfg#L35 @henryiii any ideas?

Reply:
it's listed for python<3.8
GHA uses python 3.9 in the build that's linked.

Reply:
oohh i should be importing from `cibuildwheel.typing`. will fix

Reply:
Ok, that should work now.

Reply:
All good now !